OrthoEvol/Orthologs/Blast/blast.py

```diff
@@ -494,6 +495,7 @@ def __init__(self, project="orthology-gpcr", method=1, template=None,
         super().__init__(project=project, method=method, template=template,
```

Comment:
(Optional) Instead of explicitly setting these variables you could just use **\*\*kwargs**, which encompasses the parameters of `OrthoBlastN`:

```python
super().__init__(ref_species=self.ref_species,
                         taxon_file=self.taxon_file,
                         post_blast=self.__post_blast,
                         project_path=self.project_path,
                         proj_mana=self.proj_mana, 
                         **kwargs)
```

Reply:
Good point here.